BaseApp/COD53080.TXT

```diff
@@ -0,0 +1,39 @@
+OBJECT Codeunit 53080 DotNet_MD5 Crypto Service
```

Comment:
I think we've retired support for MD5 due to vulnarabilities. Is this needed?

Reply:
We need to be able to encrypt and decrypt using a key and do the same thing in C#.  MD5 is not required, just a method that works and we can trust.

As we send passwords inside a Json to Azure Function they are encrypted with a key and the Azure Function has the key and the function to decrypt the password before passing to the actual Soap service.